github.go

```diff
@@ -198,8 +199,66 @@ func (m *GithubClient) PostComment(prNumber, comment string) error {
 	return err
 }
 
+// GetChangedFiles ...
+func (m *GithubClient) GetChangedFiles(prNumber string, commitRef string) ([]ChangedFileObject, error) {
+	pr, err := strconv.Atoi(prNumber)
+	if err != nil {
+		return nil, fmt.Errorf("failed to convert pull request number to int: %s", err)
+	}
+
+	var cfo []ChangedFileObject
+
+	var filequery struct {
+		Repository struct {
+			PullRequest struct {
+				Files struct {
+					Edges []struct {
+						Node struct {
+							ChangedFileObject
+						}
+					} `graphql:"edges"`
+					PageInfo struct {
+						PageInfoObject
```

Comment:
Instead of a separate `PageInfoObject` added to `models.go`, you could just use an inline struct as done in other cases where pagination is implemented. I.e., just inline the struct:

```go
PageInfo struct {
    EndCursor   githubv4.String
    HasNextPage bool
}
```

Reply:
I was thinking about reusability for adding pagination on commits at one point if needed

Reply:
Will do what is used in other places for pagination